.github/workflows/reusable_cve_scan.yml

```
@@ -0,0 +1,211 @@
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          submodules: recursive
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Download Docker Image
+        run: |
+          curl -L "${{ inputs.image_url }}" -o image.tar.gz
+
+      - name: Load into Docker
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:3:32: Double quote to prevent globbing and word splitting

(shellcheck)